cmd/config/init/output.go

```diff
@@ -53,3 +57,31 @@ func addCommasToNum(number string) string {
 	}
 	return result.String()
 }
+
+type OutputHandler struct {
```

Comment:
I think it makes more sense that all functions in this file which omit output should be a member of the OutputHandler class otherwise even if currently there is no problem with the functions printing output. makes it more clean IMO. 

Another option is to remove those methods from this package. 

Either way I think it makes more sense that if you have an output handler it should handle all output related command in the package.